### DISCUSSION TITLE
How do i fix thi

### DISCUSSION THREAD

**Discussion by Baconisyesssepic** [jagrosh/MusicBot#1581] [Questions]
How could i fix this error "🚫 The eval engine provided in the config (Nashorn) doesn't exist. This could be due to an invalid engine name, or the engine not existing in your version of java (21.0.3)."

**Comment by jagrosh** [comment]
JMusicBot is built with Java 11, and the `Nashorn` engine is available in Java versions up to 14. You can either install a different version of Java (between 11 and 14), or set the `evalengine` field in your config file to something that is supported on Java 21 (no other engines have been thoroughly tested, but the `Rhino` engine might work). An example config with the engine set to `Nashorn` is available [here](https://jmusicbot.com/config/).
